packages/react-avatar/src/components/AvatarBadge/useAvatarBadgeStyles.ts

```diff
@@ -0,0 +1,109 @@
+import { ax, makeStaticStyles, makeStyles } from '@fluentui/react-make-styles';
+import { AvatarBadgeState } from './AvatarBadge.types';
+
+const useStaticStyles = makeStaticStyles({
+  ':root': {
+    '--avatar-badge-size': 'var(--avatar-badge-size-small)',
+    '--avatar-badge-size-smallest': '6px',
+    '--avatar-badge-size-smaller': '8px',
+    '--avatar-badge-size-small': '10px',
+    '--avatar-badge-size-medium': '16px',
+    '--avatar-badge-size-large': '20px',
+    '--avatar-badge-size-larger': '24px',
+    '--avatar-badge-size-largest': '32px',
+
+    '--avatar-badge-borderRadius': '9999px',
+
+    // TODO this should match the parent's background color
+    '--avatar-badge-glowColor': 'var(--siteVariables-colors-white, white)',
+
+    '--avatar-badge-color': 'var(--siteVariables-colors-grey-350, rgb(151, 149, 147))',
+    '--avatar-badge-iconColor': 'var(--siteVariables-colors-white, white)',
+    '--avatar-badge-error-color': 'var(--siteVariables-colors-red-400, rgb(196, 49, 75))',
+    '--avatar-badge-error-iconColor': 'var(--siteVariables-colors-white, white)',
+    '--avatar-badge-info-color': 'var(--siteVariables-colors-brand500, rgb(151, 149, 147))',
+    '--avatar-badge-info-iconColor': 'var(--siteVariables-colors-white, white)',
+    '--avatar-badge-success-color': 'var(--siteVariables-colors-green200, rgb(146, 195, 83))',
+    '--avatar-badge-success-iconColor': 'var(--siteVariables-colors-white, white)',
+    '--avatar-badge-warning-color': 'var(--siteVariables-colors-yellow-400, rgb(248, 210, 42))',
+    '--avatar-badge-warning-iconColor': 'var(--siteVariables-colors-white, white)',
```

Comment:
I have been also considering camel case here (`--avatarBadge-warning-iconColor`). Open to suggestions.

Reply:
The AvatarBadge component is going to be deleted as soon as I have time to hook up the new PresenceBadge to Avatar, so I wouldn't worry too much about it.